### PR TITLE
Added icons to the IOU payments mode

### LINKED ISSUE #3791: [HOLD for payment on July 8] Add icons to the IOU payment methods
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!

Upwork job: https://www.upwork.com/jobs/~01b50df711834398c8
___

## Deliverables 

- Add icons to all of the IOU payment methods in the payment method selector within the IOUDetails modal

**Reproduction Steps**
1. Click the `+` icon and select `Request Money`
2. Enter an amount and click `next`
3. Select an attendee and then click `confirm`
4. Log-in to the account you just sent the request to 
5. Click the badge icon on the chat row in the LHN to access the IOUDetails modal 
6. Click the `v` on the confirmation button to access the payment method selector

## Platform:
All Platforms

**Version Number:** `v1.0.74-0`
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

image

CC: @shawnborton can you post the assets for this? 
__________________________________
[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

$ fixes #3791

### Tests | QA Steps

    1. Click the + icon and select Request Money
    2. Enter an amount and click next
    3. Select an attendee and then click confirm
    4. Log-in to the account you just sent the request to
    5. Click the badge icon on the chat row in the LHN to access the IOUDetails modal
    6. Click the v on the confirmation button to access the payment method selector.
    7. You should see icons with all methods.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web | Desktop

image

#### Mobile Web

image

#### Android

Screenshot 2021-07-01 16:54:21

### CODE REVIEW

File: src/components/Icon/Expensicons.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import ChatBubble from '../../../assets/images/chatbubble.svg';
 import Checkmark from '../../../assets/images/checkmark.svg';
 import Clipboard from '../../../assets/images/clipboard.svg';
 import Close from '../../../assets/images/close.svg';
+import Cash from '../../../assets/images/cash.svg';
```
Comment (MonilBhavsar) [inline src/components/Icon/Expensicons.js:13]: Looks like we keep this in dictionary order. Same for `Paypal`.
Not related to this PR if you can also place `NewWorkspace` and `ExpensifyCard` in correct place so we maintain an order.

Comment (parasharrajat) [reply src/components/Icon/Expensicons.js:13]: Yup

**Comment by parasharrajat** [comment]
@MonilBhavsar updated.

File: src/components/Icon/Expensicons.js (hunk 1 above)
Comment (MonilBhavsar) [reply src/components/Icon/Expensicons.js:13]: I think you forgot to change `Cash` and `Paypal`  😄

Comment (parasharrajat) [reply src/components/Icon/Expensicons.js]: Check now.

**Review by MonilBhavsar — APPROVED** [review]

**Comment by roryabraham** [comment]
Unfortunately the deploy comments are not working right now. This was deployed to staging yesterday.

**Pull request merged**